pkg/controllers/repo_config.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/DopplerHQ/cli/pkg/models"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"gopkg.in/yaml.v3"
+)
+
+// repoConfigFileName (doppler.yaml)
+var repoConfigFileName string
+
+// RepoConfigFile (e.g. ./doppler.yaml)
+var RepoConfigFile string
+
+func init() {
+	repoConfigFileName = "doppler.yaml"
+	RepoConfigFile = filepath.Join("./", repoConfigFileName)
+}
```

Comment:
If we're setting default values in `init()`, can we just set the values when the variables are declared?

```suggestion
const repoConfigFileName := "doppler.yaml"

// RepoConfigFile (e.g. ./doppler.yaml)
RepoConfigFile := filepath.Join("./", repoConfigFileName)
```